samplingsimulatorpy/generate_virtual_pop.py

```diff
@@ -34,7 +33,8 @@ def generate_virtual_pop(size, population_name, distribution_func, *para):
 
     Examples
     --------
-    >>> from samplingsimulatorpy import generate_virtual_pop
+    >>> from samplingsimulatorpy.generate_virtual_pop
+                                                import generate_virtual_pop
```

Comment:
Is this indentation correct?

Reply:
I'm afraid I have to do this, otherwise flake8 will complain about it

Reply:
I will make them on the same vertical line. Thanks!

Reply:
Done!